src/scss/main.scss

```diff
@@ -858,16 +858,31 @@ the form-control fails to do that so we do the border ourselves */
         min-width: 0 !important;
     }
 
+    /* don't print all that stuff */
+    nav,
+    footer,
+    #menu,
+    #filesdiv,
+    #comment_container,
+    #withSelected,
+    .item-info > .item-checkbox,
+    .item-info > .clickable,
+    .real-container > span,
+    .real-container > hr,
+    .real-container > form,
+    .elab-tooltip,
+    .col-md-10,
+    .col-md-2,
+    .smallgray,
+    .page-buttons {
```

Comment:
so this should be replaced by previousPage and nextPage, no?

Reply:
No, it hides the parent div of the buttons.

Reply:
oh yeah, didn't see it was still there from the "commit view"